packages/miniflare/src/workers/core/entry.worker.ts

```diff
@@ -351,6 +355,156 @@ async function handleScheduled(
 	});
 }
 
+function createForwardableEmailMessage(
+	from: string,
+	to: string,
+	raw: ReadableStream<Uint8Array<ArrayBufferLike>>,
+	headers: Headers,
+	setReject: (reason: string) => void,
+	forward: (rcptTo: string, headers?: Headers) => Promise<void>,
+	reply: (message: EmailMessage) => Promise<void>
+) {
+	return {
+		from,
+		to,
+		raw,
+		headers: headers,
+		rawSize: Number(headers.get("Content-Length")),
+		setReject,
+		forward,
+		reply,
+	} satisfies ForwardableEmailMessage;
+}
+
+async function handleEmail(
+	params: URLSearchParams,
+	request: Request,
+	service: Fetcher,
+	env: Env,
+	_ctx: ExecutionContext
+): Promise<Response> {
+	// `from` and `to` are the equivalent ones to the SMTP RCPT commands (not the ones in the email message itself)
```

Comment:
"SMTP RCPT commands" I dunno what this means. Can we add a link or ref to docs to help readers have more context here